### PR TITLE
updated filters

### PR DESCRIPTION
**Pull request by vidhi-thakur**
- added radio button to sort from price
- added checked attribute for inputs and updating value to false when the filter clears

### CODE REVIEW

File: src/components/sidebar/Sidebar.js [hunk 1]
```diff
@@ -9,44 +9,66 @@ const filters = [
...
           <input type="range" id="price-range" name="price-range" className="price-range" min="50" max="200" />
         </div>
       </section>
 
+      <section>
+        <h3 className="mb-05">Sort By</h3>
+        <div className='sort-container'>
+          {sortBy.map(option => <div key={option} onClick={() => setFilters((filters) => updateSort(filters, option))} className="sort-input">
+            <input checked={isChecked.includes(option)} type="radio" id={option} name="sort-by" className="sort-range" value={option} />
+            <label className='cp' for={option}>{option}</label>
```
Comment (msdeshmukh009) [inline src/components/sidebar/Sidebar.js]: If I am not wrong in react we have an attribute called `htmlFor` you can use that instead of `for`

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking fine and filters are working as expected.

File: src/components/filterType/FilterType.js [hunk 2]
```diff
@@ -3,20 +3,16 @@ import React from 'react';
 // css
 import './FilterType.css'
 
-function FilterType({ heading, inputs, setFilters }) {
+function FilterType({ heading, inputs, setFilters, isChecked, setIsChecked }) {
     const updateCategory = (filters, input) => {
```
Comment (bharati-21) [inline src/components/filterType/FilterType.js]: Could use a more readable name. By input, it's difficult to understand what kind of value it holds.

File: src/components/filterType/FilterType.js [hunk 3]
```diff
@@ -3,20 +3,16 @@ import React from 'react';
...
         }
     }
-    const updateSort = (filters, input) => {
-        return {
-            ...filters,
-            sort: filters.sort.includes(input) ? filters.sort.filter(data => data !== input) : [input]
-        }
-    }
     const updateRating = (filters, input) => {
+        !filters.rating.includes(input) ? setIsChecked([...isChecked, input]) : setIsChecked(isChecked.filter(data => data !== input))
```
Comment (bharati-21) [inline src/components/filterType/FilterType.js]: Suggestion - for filters and rating if you are using the same function, then you can call this function seperately in both the updateRating and updateFilter functions.

File: src/components/sidebar/Sidebar.js [hunk 4]
```diff
@@ -9,44 +9,66 @@ const filters = [
     type: "Category",
     list: ["All items", "Handmade", "Vintage"]
```
Comment (bharati-21) [inline src/components/sidebar/Sidebar.js:10]: Suggestion- if you are using mockbee and the categories data from mockbee, you could fetch that and use here. This way any change in categories, you need not make the change seprately in home page (where you render the categories) and sidebar/ product listing page.

**Review by bharati-21 — APPROVED** [review]
- The components have been broken down well.
- You have use a separate filter component for rating/ categories functionalties which was very nice.

Looks good to be merged.

File: src/components/sidebar/Sidebar.js (hunk 4 above)
Comment (vidhi-thakur) [reply src/components/sidebar/Sidebar.js:10]: yes, I do plan to update it in the later iteration along with other backend integration

File: src/components/filterType/FilterType.js (hunk 3 above)
Comment (vidhi-thakur) [reply src/components/filterType/FilterType.js]: can be done, thanks for suggestion

File: src/components/filterType/FilterType.js (hunk 2 above)
Comment (vidhi-thakur) [reply src/components/filterType/FilterType.js]: will update

**Pull request merged**